code/game/gamemodes/scoreboard.dm

```diff
@@ -438,6 +438,7 @@
 		dat += "<B>Station Deficit:</B> [num2text(profit,50)]<BR>"}*/
 	dat += {"<B>Food Eaten:</b> [score["foodeaten"]]<BR>
 	<B>Times a Clown was Abused:</B> [score["clownabuse"]]<BR>
+	<B>Number of slips: </B> [score["slips"]]<BR>
```

Comment:
This should probably be more specific, and should definitely be title case like (almost) all of the others